### ISSUE TITLE
status scrolls forever

### ISSUE THREAD

**Issue by tomchiverton** [serverless/serverless#10887]
### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

status line does not overwrite, scrolls forever

### Service configuration (serverless.yml) content

```yaml
N/A
```

### Command name and used flags

sls deploy

### Command output

```shell
⠴ Uploading (614.05 kB) (3s)

⠦ Uploading (614.05 kB) (3s)

⠧ Uploading (614.05 kB) (3s)

⠧ Uploading (614.05 kB) (4s)

⠇ Uploading (614.05 kB) (4s)

⠏ Uploading (614.05 kB) (4s)

⠏ Uploading (614.05 kB) (4s)

⠋ Uploading (614.05 kB) (4s)

⠙ Uploading (614.05 kB) (4s)
```

### Environment information

```shell
Running "serverless" from node_modules
Framework Core: 3.8.0 (local) 3.7.5 (global)
Plugin: 6.1.6
SDK: 4.3.2
```

**Comment by mnapoli** [comment]
Hi, could you let us know how you run `serverless deploy`?

Is that running in CI/CD or a terminal? Which terminal would that be? Which OS?

It looks like the command is running in interactive mode, but the terminal is not interactive.

**Comment by tomchiverton** [comment]
Manually by hand
Konsole 19.12.3
Ubuntu 20.04.4 LTS

There are other bug reports in the forum : https://forum.serverless.com/t/serverless-command-line-status-line-keeps-scrolling/16718

**Comment by tomchiverton** [comment]
```
$ env|grep -i term
COLORTERM=truecolor
TERM=xterm-256color
```

**Comment by medikoo** [comment]
I've installed locally Ubuntu 20.04 to reproduce. still, I've stumbled on another (possibly related) very weird Node.js related issue:

https://stackoverflow.com/questions/71576027/node-js-on-ubuntu-chilld-process-with-piped-stdio-does-not-emit-stdout-and-std

Can you confirm you observe the same? If you have any explanation of that I'll be grateful

**Comment by tomchiverton** [comment]
@medikoo I've no idea how child process stdio working or not has anything to do with this.

**Comment by medikoo** [comment]
@tomchiverton I've figured above issue on Ubuntu (related to how its Snap dependency manager works).

I should be able to look into this issue this week

**Comment by jkrjo** [comment]
I can confirm exactly the same issue with serverless 3.10.0 and konsole 19.12.3 from Ubuntu 20.04 (node installed from apt, no snap used in any context).

However, this is not reproducible with serverless 3.10.0 and konsole 20.12.3 running on Debian 11. So looks like something specific to terminal handling in some konsole (KDE terminal) variants.

**Comment by medikoo** [comment]
I was able to reproduce it, and fixed it in responsible dependency (`cli-color`, fix was released with v2.0.2). Therefore if you ensure to have it updated, problem should be gone (cause was relying on ANSI codes which apparently are not supported in some environments).

Next version of the Framework will guarantee that latest version of `cli-color` is installed internally

**Comment by tomchiverton** [comment]
Yup, seems fixed on a new install of latest.
